.github/maintainers_guide.md

```diff
@@ -43,23 +43,20 @@ file to import/export all of the defined functions.
 
 #### Instructions
 
-1. First, you'll need to grab a payload from `functions.list`, it must contain
-   all the connectors we want to generate. Place the response in a
-   `functions.json` file inside of this `scripts` directory.
-
-2. From the project level directory, run the generate script:
+1. From the project level directory, run the generate script:
 
    ```sh
    ./scripts/generate
    ```
 
-3. This will output something like the following:
+   This will fetch the latest connector data from Slack and generate the
+   typescript files.
+
+2. This will output something like the following:
 
    ```txt
    Cleaning folder directory
    ...
-   Generated 19 Connectors with their unit tests
-   Updated functions module export
    Formatting Slack function files...
    Linting Slack function files...
    ```
```

Comment:
As part of releasing, is it expected to run the generation script? If so, maybe we should update the release steps below? I.e. run generation, commit the changes, push up, etc.

Reply:
I think we should keep this separate for now, in the near future `run generation, commit the changes, push up, etc.` may be automated in a GitHub action and all the maintainers will have to do to "release" follow the current `Releasing steps`